.ci/aws/Jenkinsfile

```diff
@@ -12,7 +12,7 @@ def get_portafiducia_download_path() {
                 script: "aws sts get-caller-identity --query Account --output text | tr -dc 0-9",
                 returnStdout: true
               )
-    return "s3://libfabric-ci-$AWS_ACCOUNT_ID-us-west-2/portafiducia/portafiducia.tar.gz"
+    return "s3://vidsouza-testing/portafiducia.tar.gz"
```

Comment:
This doesn't seem like something we should commit :).

Reply:
Agreed. This commit will not be merged. This only a testing commit. I created this as draft so that this can't be merged.